includes/definitions/dlink-dgs1250.yaml

```diff
@@ -0,0 +1,15 @@
+os: dlink-dgs1250
+text: 'D-Link Switch'
+type: network
+icon: dlink
+#group: dlink
+mib_dir: dlink_dgs1250
```

Comment:
please use the "mib_dir: dlink" to use the existing MIB files. If needed, upgrade them (or add missing ones)

Reply:
hi @PipoCanaja 
separated OS named dlink-dgs1250 was added because MIB files needed for this switch are not equal with existing MIB files. OIDs are resolved/translated differently. I could NOT add these MIB files in existing DLINK folder, because they have same name

Reply:
Could you show us the differences between existing and new ones ? Depending how bad it is, we could solve it in a single OS.

Reply:
> Could you show us the differences between existing and new ones ? Depending how bad it is, we could solve it in a single OS.

hope it will be enough
part of sensor discovery with standard dlink mibs
```
Caching data: os processors SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OteQUSa' '-Ih' '-m' 'DLINKSW-ENTITY-EXT-MIB:DLINKSW-DDM-MIB' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/dlink' 'udp:HOSTNAME:5001' 'DLINKSW-ENTITY-EXT-MIB::dEntityExtCpuUtilTable']
DLINKSW-ENTITY-EXT-MIB::dEntityExtCpuUtilTable = No Such Instance currently exists at this OID  
  
Invalid snmp_walk() data = DLINKSW-ENTITY-EXT-MIB::dEntityExtCpuUtilTable = No Such Instance currently exists at this OID  
mempools SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OteQUSa' '-Ih' '-m' 'DLINKSW-ENTITY-EXT-MIB:DLINKSW-DDM-MIB' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/dlink' 'udp:HOSTNAME:5001' 'DLINKSW-ENTITY-EXT-MIB::dEntityExtMemoryUtilTable']
DLINKSW-ENTITY-EXT-MIB::dEntityExtMemoryUtilTable = No Such Instance currently exists at this OID  
  
Invalid snmp_walk() data = DLINKSW-ENTITY-EXT-MIB::dEntityExtMemoryUtilTable = No Such Instance currently exists at this OID  
sensors SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OteQUsa' '-Ih' '-m' 'DLINKSW-ENTITY-EXT-MIB:DLINKSW-DDM-MIB' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/dlink' 'udp:HOSTNAME:5001' 'dEntityExtEnvTempTable']
dEntityExtEnvTempTable = No Such Instance currently exists at this OID  
  
Invalid snmp_walk() data = dEntityExtEnvTempTable = No Such Instance currently exists at this OID  
SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OteQUsa' '-Ih' '-m' 'DLINKSW-ENTITY-EXT-MIB:DLINKSW-DDM-MIB' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/dlink' 'udp:HOSTNAME:5001' 'dDdmIfInfoTable']
dDdmIfInfoTable = No Such Instance currently exists at this OID  
  
Invalid snmp_walk() data = dDdmIfInfoTable = No Such Instance currently exists at this OID  
SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OteQUsa' '-Ih' '-m' 'DLINKSW-ENTITY-EXT-MIB:DLINKSW-DDM-MIB' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/dlink' 'udp:HOSTNAME:5001' 'dEntityExtEnvFanTable']
dEntityExtEnvFanTable = No Such Instance currently exists at this OID  
  
Invalid snmp_walk() data = dEntityExtEnvFanTable = No Such Instance currently exists at this OID  
SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OteQUsa' '-Ih' '-m' 'DLINKSW-ENTITY-EXT-MIB:DLINKSW-DDM-MIB' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/dlink' 'udp:HOSTNAME:5001' 'dEntityExtEnvPowerTable']
dEntityExtEnvPowerTable = No Such Instance currently exists at this OID
```